python/hsfs/training_dataset.py

```diff
@@ -157,6 +161,8 @@ def __init__(
                 else extra_filter
             )
         else:
+            self._start_time = event_start_time
```

Comment:
`event_start_time` and `event_end_time` do not need to transformed if they are returned from backend.